compose.yaml

```diff
@@ -21,8 +24,6 @@ services:
       - 8443:8443
   mongodb:
     image: mongo:4
-    ports:
-      - "27017:27017"
```

Comment:
Don't actually need to expose the mongodb port to the host system - it just pollutes ports on the host. All ports are accessible in the internal Docker network.